app/lib/dags/experimental.json.erb

```diff
@@ -38,7 +38,7 @@
       "taxid_locations_combined.json"
     ],
     "alignment_viz_out": ["align_viz.summary"],
-    "srst2_out": ["output.log", "output__genes__ARGannot_r2__results.txt", "output__fullgenes__ARGannot_r2__results.txt", "amr_processed_results.csv", "amr_summary_results.csv"],
+    "srst2_out": ["out.log", "out__genes__ARGannot_r2__results.txt", "out__fullgenes__ARGannot_r2__results.txt", "amr_processed_results.csv", "amr_summary_results.csv", "output__.ARGannot_r2.sorted.bam"],
```

Comment:
Out of curiosity, what's the rollout plan? Are you going to push to prod and push the dag change to master at the same time?

Reply:
I notice that this file still contains the guards from a past rollout (`<% if attribute_dict[:pipeline_version].to_f >= 3.6 %>`). That approach seems safer to me, but if you talked with Charles and feel that the coordinated rollout is better, please feel free.

Reply:
That's the plan, yeah. I could deploy it to staging for a short bit, as long as people are aware that samples won't run unless they specify the `lvreynoso/amr_reads` pipeline branch.